README.md

```diff
@@ -186,14 +186,15 @@ const HiddenItemWrappedTwice = ()=> <HiddenItemWrapped />
 
 `HiddenItem` accepts:
 
-| prop and type              | description                                                 | note |
-| -------------------------- | ----------------------------------------------------------- | ---- |
-| title: string              | title for the button, required                              |      |
-| style?: ViewStyleProp      | style to apply to the touchable element that wraps the text |      |
-| titleStyle?: ViewStyleProp | style to apply to the text                                  |      |
-| onPress: ?() => any        | function to call on press                                   |      |
-| testID?: string            | testID to locate view in e2e tests                          |      |
-| disabled?: boolean         |                                                             |      |
+| prop and type              | description                                                        | note     |
+| -------------------------- | ------------------------------------------------------------------ | -------- |
+| title: string              | title for the button, required                                     |          |
+| style?: ViewStyleProp      | style to apply to the touchable element that wraps the text        |          |
+| titleStyle?: ViewStyleProp | style to apply to the text                                         |          |
+| onPress: ?() => any        | function to call on press                                          |          |
+| testID?: string            | testID to locate view in e2e tests                                 |          |
+| disabled?: boolean         | disabled 'item' is greyed out and `onPress` is not called on touch |          |
+| destructive?: boolean      | flag specifying whether this item is destructive                   | iOS only |
```

Comment:
can you please add this to the TS typings file too? Thanks

Reply:
Done